starport/services/scaffolder/patch.go

```diff
@@ -9,6 +9,26 @@ import (
 	modulecreate "github.com/tendermint/starport/starport/templates/module/create"
 )
 
+// supportSimulation checks if module_simulation.go exists
+// appends the generator to create the file if it doesn't
+func supportSimulation(
```

Comment:
Maybe we can also create `app/simulation_test.go` if it doesn't exist

Reply:
this is the current behavior, the `xgenny.Box` do it under the hood

Reply:
I mean the test file `simulation_test.go` under `app`. It is not created if the app has been scaffolded with a previous version of Starport, for me at least

I may see that we must do modification in `app.go` so maybe this can be handled in another PR

Reply:
yes, I agree, but as you say, only putting `simulation_test.go` file doesn't work the simapp tests, we need to make modifications into the `app.go` and `x/{{module}}/module.go`. But the old chains still work with this starport version